servatrice/src/serversocketinterface.cpp

```diff
@@ -1000,3 +1002,55 @@ Response::ResponseCode ServerSocketInterface::cmdReloadConfig(const Command_Relo
     settingsCache->sync();
     return Response::RespOk;
 }
+
+Response::ResponseCode ServerSocketInterface::cmdAdjustMod(const Command_AdjustMod &cmd, ResponseContainer & /*rc*/) {
+
+    QString adminName = QString::fromStdString(cmd.admin_name());
+    QString userName = QString::fromStdString(cmd.user_name());
+
+    if (cmd.should_be_mod()) {
+        logDebugMessage("Received admin command: promote user");
+        QSqlQuery *query = sqlInterface->prepareQuery(
+                "update {prefix}_users set admin = :adminlevel where name = :username");
+        query->bindValue(":adminlevel", "2");
```

Comment:
Are we not able to just say `set admin = 2 where name...`? If so, i don't see a reason for the bind. If not, then this is fine.